common.mk

```diff
@@ -141,22 +141,18 @@ run-binary-fast: $(sim)
 #########################################################################################
 # helper rules to run simulator with as much debug info as possible
 #########################################################################################
-run-binary-debug: $(sim_debug)
```

Comment:
You need to add the output_dir dependency to the other run-binary rules as well.